pr_watcher_notifier/templates/email_body.txt

```diff
@@ -0,0 +1,6 @@
+Hi,
+
+Pull request #{{ number }} ({{ pr_url }}) against the {{ repo }} repository has been {{ action }}.
```

Comment:
The current code uses only the first match to make a decision because I thought that it will be efficient. Thinking about it now, I don't think the performance difference is going to matter unless we have thousands of files modified. So I'll make the change.